radis/test/lbl/test_calc.py

```diff
@@ -148,32 +149,39 @@ def test_calc_spectrum(verbose=True, plot=True, warnings=True, *args, **kwargs):
     #                           0.17110649,  0.15403513,  0.13376804,  0.11932659,  0.10882006,
     #                           0.11112725,  0.0458288 ,  0.00247956,  0.00144128])
     # Updated again in RADIS 0.9.20 (02/09/19) with switch to tabulated Q(Tref)
+    # I_ref = np.array([         0.29048064, 0.29743104, 0.32955513, 0.32047172, 0.20688813,
+    #                           0.19210952, 0.20148265, 0.17330909, 0.17213373, 0.15887159,
+    #                           0.17106096, 0.15400039, 0.13374285, 0.11930822, 0.10880631,
+    #                           0.11111394, 0.04582291, 0.00247955, 0.00144128])
+    # Updated again on (05/08/20) with implementation of optimized weights:
     I_ref = np.array(
         [
-            0.29048064,
-            0.29743104,
-            0.32955513,
-            0.32047172,
-            0.20688813,
-            0.19210952,
-            0.20148265,
-            0.17330909,
-            0.17213373,
-            0.15887159,
-            0.17106096,
-            0.15400039,
-            0.13374285,
-            0.11930822,
-            0.10880631,
-            0.11111394,
-            0.04582291,
-            0.00247955,
-            0.00144128,
+            0.29043204,
```

Comment:
That's explained here:
>The previous weights were hard coded to be "minimized-RMS for pure Gaussian/Lorentzian", and have been removed since they perform worse than simple weights. This means that the default option had to be changed, and it is set to optimized weights. This also changed the hard-coded values in the test in test_calc.py, which have been updated.

>The tests in test_calc.py now use the simple weights (optimized_weights = False), so that the optimized weights can be replaced with even more optimized weights in the future -- should they ever be found -- without compromising the test.

Reply:
Sounds good, I didn't understand the difference between pure Gaussian-Lorentzian and simple weights !!